### PR TITLE
Feature: add candidate-status & candidate-resume tests

### PR DESCRIPTION
**Pull request by sajidaqadomi**
candidate-status-video
https://1drv.ms/v/s!AlQUJbHNeZSJkAhTsdtsKvepe1nf?e=71ZQMz

candidate-resume-video
https://1drv.ms/v/s!AlQUJbHNeZSJkAftPnhBtU3g4WAE?e=U8fOuY

Screenshot 2023-11-12 235753

### CODE REVIEW

File: cypress/e2e/orange/recruitment/candidate-status/candidate-status-spec.cy.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+Then(
+  "The candidate should have the appropriate action buttons:",
+  (actions: any) => {
+    const actionButtons = actions.rawTable[0];
+
+    candidateProfilePage.verifyActionBtn(actionButtons);
+  }
+);
+
+After(() => {
```
Comment (Rami-Ilaiwi) [inline cypress/e2e/orange/recruitment/candidate-status/candidate-status-spec.cy.ts]: keep using `afterEach` instead of `After`

**Review by Rami-Ilaiwi — COMMENTED** [review]
thank you Sajida! this is so good!

**Comment by sajidaqadomi** [comment]
> thank you Sajida! this is so good!

Thanks Rami

File: cypress/e2e/orange/recruitment/candidate-status/candidate-status-spec.cy.ts (hunk 1 above)
Comment (sajidaqadomi) [reply cypress/e2e/orange/recruitment/candidate-status/candidate-status-spec.cy.ts]: updated

**Pull request merged**